src/pages/guides/using/security_general.md

```diff
@@ -40,4 +40,9 @@ For web actions, see [Securing Web Actions](securing_web_actions.md).
 
 Due to the shared nature of any serverless system, it is necessary to review transport security to your function and how your function communicates with the world.
 
-All communications to the Runtime cluster are secured by HTTPS. Communications from your functions to any other service should also use HTTPS or some other secure channel whenever possible. While there is a network partition and no function should be able to see or route traffic directly to another, it is always best to also secure your communication channel as well.
\ No newline at end of file
+All communications to the Runtime cluster are secured by HTTPS. Communications from your functions to any other service should also use HTTPS or some other secure channel whenever possible. While there is a network partition and no function should be able to see or route traffic directly to another, it is always best to also secure your communication channel as well.
+
+## Allowed Egress Ports
+Actions can call any external IP, however the number of ports that can be used, is limited. This is the list of ports that can be used for external API calls:
```

Comment:
> _...the **number** of ports..._
I feel like this should be worded as _the **particular**  ports_ (or even _specific_)